ansibleflow/venv.py

```diff
@@ -38,7 +39,9 @@ def execute_under_env(command, os_env=None):
         proc.stdin.write(long_cmd.encode('utf-8'))
         proc.stdin.flush()
 
-        proc.wait()
+        returncode = proc.wait()
+        if returncode:
+            sys.exit(returncode)
```

Comment:
Yeah basically any non-zero response would be good to terminate processing on, to halt Jenkins from proceeding to the next job in the chain for example.